src/pages/[platform]/prev/start/project-setup/index.mdx

```diff
@@ -1,37 +1,230 @@
 import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
-import { getChildPageNodes } from '@/utils/getChildPageNodes';
 
 export const meta = {
-  title: 'Project setup',
-  description: 'Project setup',
+  title: 'Amplify project setup',
+  description:
+    'Set up Amplify prerequisites and a project.',
   platforms: [
-    'android',
     'angular',
-    'flutter',
     'javascript',
     'nextjs',
     'react',
-    'react-native',
+    'vue',
     'swift',
-    'vue'
-  ],
-  route: '/[platform]/prev/start/project-setup'
+    'android',
+    'flutter',
+    'react-native'
+  ]
 };
 
 export const getStaticPaths = async () => {
   return getCustomStaticPath(meta.platforms);
 };
 
 export function getStaticProps(context) {
-  const childPageNodes = getChildPageNodes(meta.route);
   return {
     props: {
       platform: context.params.platform,
-      meta,
       showBreadcrumbs: false,
-      childPageNodes
+      meta
     }
   };
 }
 
-<Overview childPageNodes={props.childPageNodes} />
+<Callout info>
+
+We recommend that you use the [latest version for project setup](/[platform]/start/project-setup/) for your chosen framework or language. 
+
+This page is a reference for those who need the previous version and includes all project setup content for that version. Clicking the navigation menu will take you back to the current version by default.
```

Comment:
For accessibility purposes, we should avoid "clicking." Change to "Selecting the navigation menu..."